openlibrary/plugins/openlibrary/js/carousel/Carousel.js

```diff
@@ -72,15 +73,18 @@ const Carousel = {
             slidesToScroll: a,
             responsive: responsive_settings
         });
-
+        //This loads in i18n strings from a hidden input element, generated in the books/custom_carousel.html template.
+        const i18nValues = JSON.parse($('input[name=\'carousel-i18n-strings\']').attr('value'))
         availabilityStatuses = {
-            open: {cls: 'cta-btn--available', cta: 'Read'},
-            borrow_available: {cls: 'cta-btn--available', cta: 'Borrow'},
-            borrow_unavailable: {cls: 'cta-btn--unavailable', cta: 'Join Waitlist'},
-            error: {cls: 'cta-btn--missing', cta: 'Not In Library'},
+            open: {cls: 'cta-btn--available', cta: i18n_values['open']},
+            borrow_available: {cls: 'cta-btn--available', cta: i18n_values['borrow_available']},
+            borrow_unavailable: {cls: 'cta-btn--unavailable', cta: i18n_values['borrow_unavailable']},
+            error: {cls: 'cta-btn--missing', cta: i18n_values['error']},
```

Comment:
```suggestion
            open: {cls: 'cta-btn--available', cta: i18nValues['open']},
            borrow_available: {cls: 'cta-btn--available', cta: i18nValues['borrow_available']},
            borrow_unavailable: {cls: 'cta-btn--unavailable', cta: i18nValues['borrow_unavailable']},
            error: {cls: 'cta-btn--missing', cta: i18nValues['error']},
```
`i18n_values` is no longer defined.